src/components/Order/OrderMain/OrderState/OrderInitial/OrderInitialFiat/OrderInitialFiat.js

```diff
@@ -21,20 +21,29 @@ const OrderInitial = props => {
           </b>
         </h4>
 
-          <input type="checkbox" name="checkboxTC" value="check"/>
-          <strong dangerouslySetInnerHTML={{__html: t('order.iAgreedTC')}}/>
+        <input type="checkbox" name="checkboxTC" value="check" style={{ width: '20px', height: '20px', }}
+        onClick={function () {
+            let _checkoutButton = document.getElementsByName("checkoutButton")[0];
```

Comment:
better to put this outside of the markup as a named function on the scope and then call it.

Reply:
problem is this one is inherited view https://github.com/onitsoft/nexchange-open-client-react/blob/acd5553888e1980477beeaf81385a8ed5e20bd99/src/components/Order/OrderMain/OrderState/OrderInitial/OrderInitial.js#L48
did not found a way how to define javascript in this file (tried - failed) javascript from main file is not working here.

Reply:
Ok, then just name the function, it would be easier to traceback in case of errors.

As in 
```
        onClick={function togglePayNowButton() {
```